api/features/multivariate/models.py

```diff
@@ -31,6 +31,13 @@ class MultivariateFeatureOption(
     AbstractBaseExportableModel,
     abstract_base_auditable_model_factory(["uuid"]),
 ):
+    """
+    This class holds the *value* for a given multivariate feature
+    option. This value is the same for every environment, but the
+    percent allocation is set in MultivariateFeatureStateValue
+    which varies per-environment.
+    """
```

Comment:
I decided to add this docstring when I was learning the codebase. I found it confusing that it held the value while the value class actually held the percent.